src/library/Connect/LocalEnvironmentManager.cs

```diff
@@ -226,7 +226,11 @@ CancellationToken cancellationToken
                 {
                     this._reversePortForwardManager.Start(pfInfo);
                     this._ReportProgress(Resources.ContainerPortAvailableFormat, pfInfo.Port, pfInfo.LocalPort);
+                    if (pfInfo.Port == DevHostConstants.DevHostAgent.Port) {
+                        _log.Warning($"Service to be debug uses same port '{pfInfo.Port}' as Bridge DevHostAgent service does. This can cause conflict. It's recommended to update service to use different port.");
+                    }
                 }
+                perfLogger.SetSucceeded();

```

Comment:
If you look at the library logs for a run you will see below log for LocalEnvironmentManager-StarReversePortForwarding: 
2022-12-08T21:38:51.5539847Z | Library | TRACE | Event: **LocalEnvironmentManager-StartReversePortForwarding** <json>{"properties":{"**result":"Failed**"},"metrics":{"durationInMs":8.0}}</json>

While if you look at log for LocalEnvironmentManager-StartServicePortForwardings you will see below log
2022-12-12T19:19:09.2574798Z | Library | TRACE | Event: LocalEnvironmentManager-StartServicePortForwardings <json>{"properties":{"result":"Succeeded"},"metrics":{"durationInMs":10.0}}</json>

Note it is different to the one that comes from ConnectMaangementClient.
2022-12-12T19:19:09.2621246Z | Library | TRACE | Event: ConnectManagementClient-StartServicePortForwardings <json>{"properties":{"result":"Succeeded"},"metrics":{"durationInMs":16.0}}</json>

Essentially, everytime you start a PerfLogger it will log that log line when it was out of scope (this is why there is 3 log lines, one for each start and one for the actions that starts all the port forwarding), and it initial state/result if not set is False.